docs/snippets/solid/mdx-canvas-multiple-stories.mdx.mdx

```diff
@@ -0,0 +1,39 @@
+```md
```

Comment:
@webblocksapp if you're ok with it, could you remove this snippet, as with Storybook 7 this pattern shouldn't be used. And other examples are now removed from our documentation

Reply:
Done.